articles/libraries/auth0js/v8/migration-guide.md

```diff
@@ -169,14 +180,13 @@ To switch from HS256 to RS256 for a specific client, follow these instructions:
 
 Remember that if the token is being validated anywhere else, changes might be needed there as well in order to comply.
 
-### Manually Parsing Hashes 
+### Manually Parsing Hashes
 
 If you would rather manually parse hashes, to avoid the `parseHash` method since it only works with RS256, feel free to take a look at [this example](https://github.com/auth0/auth0.js/blob/master/src/web-auth/index.js#L97) or [this one](https://github.com/auth0/auth0.js/blob/master/src/helper/qs.js#L10) to help you get started.
 
 ## Refreshing Tokens
 
-In [auth0.js v7](/libraries/auth0js/v7#refresh-token), the `renewIdToken` and `refreshToken` methods were used to renew tokens. In [auth0.js v8](/libraries/auth0js#using-renewauth-to-acquire-new-tokens), 
-refreshing tokens is now done via the `renewAuth` method. If a user is already authenticated, `renewAuth` can acquire a new token for that user.
+In [auth0.js v7](/libraries/auth0js/v7#refresh-token), the `renewIdToken` and `refreshToken` methods were used to renew tokens. In [auth0.js v8](/libraries/auth0js#using-renewauth-to-acquire-new-tokens), refreshing tokens is now done via the `renewAuth` method. If a user is already authenticated, `renewAuth` can acquire a new token for that user.
```

Comment:
- Remove `now` from `is now done`. 
- At the 1st sentence you say `renew` and at the 2nd `refresh`. I get that the meaning is the same but it's better to be consistent. 
- I'd add a short sentence at the beginning of this paragraph saying what "refreshing tokens" is